mava/systems/jax/launcher.py

```diff
@@ -117,6 +118,9 @@ def add(
 
         if self._multi_process:
             with self._program.group(name):
+                # Save the current PID to manage the termination of the process
+                node_fn.__self__.store.manager_pid = os.getpid()
```

Comment:
I believe we can't set it in the terminator component attributes because we need the pid of the launcher, I will be thankful if you have another option to deal with this pid and get it as an input to the terminator

Reply:
I will check if they have same pid if yes we can do that in the builder side

Reply:
Yes it is the same I will change it to be in `on_building_init_end`

Reply:
Done!